.github/workflows/main.yml

```
@@ -0,0 +1,90 @@
+name: Build Linux
+
+on:
+  push:
+    branches:
+      - jellyfin
+
+  pull_request:
+    branches:
+      - jellyfin
+
+  release: 
+    types:
+      - released
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build ${{ matrix.release }} ${{ matrix.arch }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        release:
+          - bullseye
+          - buster
+          - stretch
+          - groovy
+          - focal
+          - bionic
+
+        arch:
+          - amd64
+          - arm64
+          - armhf
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install make and mmv
+        run: sudo apt-get install make mmv
+
+      - name: Install Docker
+        uses: docker-practice/actions-setup-docker@v1
+
+      - name: Build
+        run: ./build ${{ matrix.release }} ${{ matrix.arch }} dist
+
+      - name: Upload Packages
+        uses: actions/upload-artifact@v2.2.3
+        with:
+          name: ${{ matrix.release }} ${{ matrix.arch }} package
+          path: dist
+
+  publish:
+    if: github.event_name == 'release' && github.event.action == 'released'
```

```suggestion
    if: ${{ github.event_name == 'release' && github.event.action == 'released' }}
```
again no strong opinion on this either